Src/WitsmlExplorer.Frontend/components/ContextMenus/LogCurvePriorityContextMenu.tsx

```diff
@@ -0,0 +1,55 @@
+import { Typography } from "@equinor/eds-core-react";
+import { MenuItem } from "@material-ui/core";
+import React, { useContext } from "react";
+import OperationContext from "../../contexts/operationContext";
+import { MousePosition } from "../../contexts/operationStateReducer";
+import { StyledMenu, preventContextMenuPropagation } from "./ContextMenu";
+import { StyledIcon } from "./ContextMenuUtils";
+
+export interface LogCurvePriorityContextMenuProps {
+  onDelete: () => void;
+  onClose: () => void;
+  position: MousePosition;
+  open: boolean;
+}
+
+export const LogCurvePriorityContextMenu = (
+  props: LogCurvePriorityContextMenuProps
+): React.ReactElement => {
+  const { onDelete, onClose, position, open } = props;
+  const { operationState } = useContext(OperationContext);
+  const { colors } = operationState;
+
+  const onClickDelete = async () => {
+    onDelete();
+    onClose();
+  };
+
+  return (
+    <StyledMenu
+      disablePortal
+      keepMounted
+      open={open}
+      onClose={onClose}
+      anchorReference="anchorPosition"
+      anchorPosition={
+        position.mouseY !== null && position.mouseX !== null
+          ? {
+              top: position.mouseY,
+              left: position.mouseX
+            }
+          : undefined
+      }
+      onContextMenu={preventContextMenuPropagation}
+      colors={colors}
+    >
+      <MenuItem key={"delete"} onClick={onClickDelete}>
```

Comment:
This is the same way you delete rows from all the other tables, so I don't think it will be a problem. Like here in the curve values view:

![image](https://github.com/equinor/witsml-explorer/assets/70512270/10378474-64a6-43a3-a4a8-cb25b98ba098)

Reply:
I know, I just found it a bit hard to find. It is fine for now :)